app/services/modalService.js

```diff
@@ -82,5 +102,26 @@ angular.module('portainer.services')
       callback: callback,
     });
   };
+
+  service.confirmContainerDeletion = function(title, text, callback) {
+    service.prompt({
+      title: title,
+      inputType: 'checkbox',
+      inputOptions: [
+        {
+          text: text,
+          value: '1'
```

Comment:
Hum I don't think so, cause bootbox generate a checkbox with value (as string) like that : 
`<input class="bootbox-input bootbox-input-checkbox" type="checkbox" value="1">`
So in result array, we have strings